src/preact/compat.js

```diff
@@ -72,6 +72,10 @@ export function forwardRef(Component) {
   return /** @type {*} */ (Forward);
 }
 
+/** @type {import('preact/compat').forwardRef} */
```

Comment:
@rcebulko: I couldn't figure out a better way to type `forwardRef` correctly from the get go, so I went with this aliasing. LMK if you can think of something more elegant

Reply:
I think you can cut out the middle-man

```javascript
/** @type {import('preact/compat').forwardRef */
export const forwardRef = function(Component) {
```

Reply:
I tried! It'll attempt to input/output of the function, which end up not being exactly right.

Reply:
Oh. I misread your idea. This works :)